drivers/syslog/Kconfig

```diff
@@ -61,6 +61,18 @@ config RAMLOG_OVERWRITE
 		Enable overwrite of circular buffer. If RAMLOG buffer overflows,
 		overwrite it from the top of buffer and always keep the latest log.
 
+config RAMLOG_KEEPLOG
```

Comment:
@PetteriAimonen please include this information in the Documentation/, maybe a separated section for RAMLOG at https://nuttx.apache.org/docs/latest/components/drivers/special/syslog.html will be nice!